pep-0654.rst

```diff
@@ -126,14 +126,24 @@ Specification
 ExceptionGroup
 --------------
 
-The new builtin exception type, ``ExceptionGroup`` is a subclass of
-``BaseException``, so it is assignable to ``Exception.__cause__`` and
-``Exception.__context__``, and can be raised and handled as any exception
-with ``raise ExceptionGroup(...)`` and ``try: ... except ExceptionGroup: ...``.
-
-Its constructor takes two positional-only parameters: a message string and a
-sequence of the nested exceptions, for example:
+We propose to add two new builtin exception types: ``ExceptionGroup`` and
+``BaseExceptionGroup``. The former is a subclass of ``Exception`` and
+the latter is a subclass of ``BaseException``. Both are assignable to
```

Comment:
Shouldn't this also explain that ExceptionGroup subclass BaseExceptionGroup? (And in which order its base classes appear?)